sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries1Id.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 1 string value that uniquely identifies a Time Series instance.
+    /// </summary>
+    /// <remarks>
```

Comment:
Also can add a remark on 

"You can also select up to three key properties as your Time Series ID. Their combination will be a composite key that represents the Time Series ID."

Reply:
This class will no longer exist in the potential design. But will add the comment to the new class that represents TimeSeriesId